omod/src/main/java/org/openmrs/module/fhir/server/FHIRRESTServer.java

```diff
@@ -68,10 +69,9 @@ protected void initialize() {
 		resourceProviders.add(new RestfulConditionResourceProvider());
 		resourceProviders.add(new RestfulDiagnosticReportResourceProvider());
 		resourceProviders.add(new RestfulMedicationResourceProvider());
-		resourceProviders.add(new RestfulMedicationRequestProvider());
+		resourceProviders.add(new RestfulMedicationRequestResourceProvider());
+		resourceProviders.add(new RestfulProcedureRequestResourceProvider());
 		resourceProviders.add(new RestfulPlanDefinitionResourceProvider());
-		//Removing composition since we now not support for it
-		//resourceProviders.add(new RestfulCompositionResourceProvider());
```

Comment:
Why did you remove this comment here and left it in SwaggerSDKGenerationTestCase.java